packages/expo-file-system/android/src/main/java/expo/modules/filesystem/FileSystemModule.java

```diff
@@ -304,7 +307,12 @@ public void deleteAsync(String uriStr, Map<String, Object> options, Promise prom
       if ("file".equals(uri.getScheme())) {
         File file = uriToFile(uri);
         if (file.exists()) {
-          FileUtils.forceDelete(file);
+          if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+            FileUtils.forceDelete(file);
+          } else {
+            // to be removed once Android SDK 25 support is dropped
```

Comment:
```suggestion
            // TODO: remove once Android SDK 25 support is dropped
```

`TODO` will show up when looking for TODOs 🙂